xml/soc-operations-self-sign-cert.xml

```diff
@@ -0,0 +1,784 @@
+        so the resulting subdomain will be: <literal>_acme-challenge.yourdomain.com</literal>
+      </screen>
+    </step>
+    <step>
+      <para>
+        Using this information, you are ready to insert this TXT record into
+        your DNS.
+      </para>
+      <note>
+        <para>
+          When setting this up for SUSE OpenStack Cloud with Crowbar, you need
+          to have your external DNS server appended to <filename>/etc/resolv.conf</filename>
+          in order to resolve as crowbar has its own internal
+          DNS. It is not enough to change it in the barclamp as you need the
+          DNS server entry to be at the top of the list in <filename>resolv.conf</filename>.
+          Crowbar returns to the default after a period of time.
+          Keep in mind that if you want to automate this step every 90 days
+          then you need to ensure the <filename>resolv.conf</filename> changes
+          every time to bypass the local crowbar DNS Server.
+        </para>
+      </note>
+    </step>
+    <step>
+      <para>
+        In order to set up TXT record in bind DNS, edit the zone file so
+        it looks like the following example:
+      </para>
+      <screen>
+        yourdomain.com.     IN NS           admin.yourdomain.com.
+        _acme-challenge.yourdomain.com. IN TXT "xxxx...your TXT value string here"
+      </screen>
+    </step>
+    <step>
+      <para>
+        Restart your named services for <literal>bind</literal>.
+      </para>
+    </step>
+    <step>
+      <para>
+        Issue the acme-challenge verification of the previous step with the
+        following command:
+      </para>
+      <screen>
+        # acme.sh --renew -d yourdomain.com
+      </screen>
+    </step>
+    <step>
+      <para>
+        If the DNS validation is okay, <filename>acme.sh</filename> issues a
+        wildcard certificate and displays the certificate and private-key path.
+        For example:
+      </para>
+      <screen>
+        Your cert is in:  /root/.acme.sh/susedojo.com/susedojo.com.cer  
+        Your cert key is in:  /root/.acme.sh/susedojo.com/susedojo.com.key  
+        v2 chain.
+        The intermediate CA cert is in:  /root/.acme.sh/susedojo.com/ca.cer  
+        And the full chain certs is in:  /root/.acme.sh/susedojo.com/fullchain.cer  
+        _on_issue_success
+      </screen>
+    </step>
+    <step>
+      <para>
+        Notice the location of your certificate and key. These are now
+        ready to be used by OpenStack Cloud.
+      </para>
+    </step>
+    <step>
+      <para>
+        To automate the process of setting up the TXT record in your DNS
+        servers and prepare it for automated validation, there is a file
+        called <filename>account.conf</filename> holds account information
+        for the DNS Provider. After exporting the authentication variables,
+        it stores them automatically after the command is executed for later use.
```

Comment:
This reads awkwardly for me. `there is a file called <filename>account.conf</filename> holds account information` ...maybe `the file account.conf holds account`...